src/NServiceBus.Core.Tests/Transports/Learning/AsyncFileTests.cs

```diff
@@ -11,7 +11,7 @@ public class AsyncFileTests
         public async Task When_file_content_larger_than_buffer_size()
         {
             var originalContent = string.Join("", Enumerable.Repeat("a#~×ψؾࢯ‽%1", 2000));
-            const string filePath = "test.txt";
```

Comment:
sorry, I meant that the old approach is not working for vstests. But we have quite a few more tests which fail due to that reason, so that's a change we should extract into a dedicated PR anyways